index.js

```diff
@@ -26,7 +26,7 @@ class Plugin extends PluginMiniAccounts {
     this._account = opts.account
     this._db = opts.db || 'machinomy_db'
     this._provider = opts.provider || 'http://localhost:8545'
-    this._minimumChannelAmount = opts.minimumChannelAmount || 100
+    this._minimumChannelAmount = opts.minimumChannelAmount || new BigNumber(100)
```

Comment:
What's the expected type of `this._minimumChannelAmount`? Should this be `opts.minimumChannelAmount || '100'`?

Reply:
I guess we could change it to `new BigNumber(opts.minimumChannelAmount || 100)`